docs/superpowers/plans/2026-04-21-myvector-log-abstraction.md

```
@@ -0,0 +1,373 @@
+**Category B — direct calls** (`my_plugin_log_message(&gplugin, LEVEL, ...)`):
+
+Each direct call is multi-line and uses one of three levels. Replace by level:
+
+| Level argument | New macro |
+|---|---|
+| `MY_INFORMATION_LEVEL` | `MYVEC_LOG_INFO` |
+| `MY_ERROR_LEVEL` | `MYVEC_LOG_ERROR` |
+| `MY_WARNING_LEVEL` | `MYVEC_LOG_WARN` |
+
+Example — before:
+```cpp
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)